### ISSUE TITLE
Custom Loss Function

### ISSUE THREAD

**Issue by zeref1234** [tensorflow/tensor2tensor#1672]
Hi,
I would like to update the loss function in universal transformer to add weights to certain translations. I am unable to figure out how to do this. Could you please help?

**Comment by lidongxing** [comment]
How about this? some progress？
